src/requests/getForecast.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  setErrorMessage
+) => {
+  let endpoint = "https://cmd-shift-weather-app.onrender.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+
+  axios
+    .get(endpoint)
+    .then((response) => {
+      setSelectedDate(response.data.forecasts[0].date);
+      setForecasts(response.data.forecasts);
+      setLocation(response.data.location);
+    })
+    .catch((error) => {
+      const { status } = error.response;
+      if (status === 404) {
+        setErrorMessage("No such town or city");
+        console.error("Location is not valid.", error);
```

Comment:
You can delete these console.errors on line 28 and line 32 as you are already displaying error messages to the user.